builder/azure/arm/config.go

```diff
@@ -151,6 +151,9 @@ type Config struct {
 
 	Comm communicator.Config `mapstructure:",squash"`
 	ctx  *interpolate.Context
+
+	//Cleanup
+	AsyncRGDelete bool `mapstructure:"async_resourcegroup_delete"`
```

Comment:
Please write a unit test to ensure you can control this value.

Reply:
added